### PR TITLE
Introducing a support for Schedule APIs

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

File: OmiseSwift/Period.swift [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        default:
+            return nil
+        }
+    }
+}
+
+extension Period.MonthlyPeriodRule.Ordinal {
+    init?(ordinalString: String) {
+        switch ordinalString.lowercased()  {
+        case "1st":
```
Comment (chakrit) [inline OmiseSwift/Period.swift:163]: Not sure but I remember @turbo said "first", "second" etc... also works.

**Review by chakrit — APPROVED** [review]
Tests :rainbow_look:

File: OmiseSwift/Period.swift (hunk 1 above)
Comment (pitiphong-p) [reply OmiseSwift/Period.swift:163]: Oh thanks for the catch. I'll confirm with him

Comment (pitiphong-p) [reply OmiseSwift/Period.swift:163]: He tole me that, `first` only be supported as a creating parameter but not gonna be return from the API. So we don't need to support it

**Review by igroomgrim — APPROVED** [review]

**Pull request merged**